### ISSUE TITLE
Use a disclosure widget for language selection

### ISSUE THREAD

**Issue by jobara** [inclusive-design/guide.inclusivedesign.ca#81]
## Is your feature request related to a problem?

In #53 we're adding localization support for English and French. This is a simple link toggle being worked on for that issue, but it would be good to use a disclosure widget, incase we have additional locales to support in the future. 

## Describe the solution you'd like

@greatislander suggested using the disclosure widget (see: https://github.com/inclusive-design/guide.inclusivedesign.ca/issues/53#issuecomment-907804477 ) and referenced this example https://www.w3.org/TR/wai-aria-practices/examples/disclosure/disclosure-navigation.html

## Additional context or notes
Depends on #53

**Comment by jobara** [comment]
I started looking into this as part of #53 but decided to move to its own issue so that we could get the localization changes in sooner. This will also be affected by changes to #54. 

In progress work can be seen in https://github.com/jobara/guide.inclusivedesign.ca/commit/0e8ff1e5e85bd6adc8ee5136bf823d939768d622

**Comment by jhung** [comment]
@jobara can you add and add/remove the `hidden` attribute from the mobile and desktop navigation containers?
